int32 n_points
geometry_msgs/Point[] path